gsa/src/gmp/commands/entities.js

```diff
@@ -92,12 +92,12 @@ class EntitiesCommand extends HttpCommand {
   }
 
   export(entities) {
-    return this.exportByIds(map(entities, entity => entity.id));
+    return this.exportByIds(map(entities.data, entity => entity.id));
```

Comment:
Same as delete. `entities` should still be an Array of ids. The calling code should be changed instead.